33Across User ID sub-module: Introduce a new supplemental ID

carlosfelix opened the pull request:
## Type of change
- [ ] Bugfix
- [X] Feature
- [ ] New bidder adapter
- [ ] Updated bidder adapter
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Enhance the 33Across User ID sub-module to support setting a third-party supplemental ID in addition to the primary ID envelope.

## Other information
https://github.com/prebid/prebid.github.io/pull/5361

patmmccann commented:
Couple concerns: 174 test failures and this will create merge conflicts with 9, can you do it on that branch?

carlosfelix commented:
> Couple concerns: 174 test failures and this will create merge conflicts with 9, can you do it on that branch?

sure, I've updated my branch with the latest 9.0 changes and changed the destination branch.  Test failures no longer showed up, although CI is still failing because some flaky browserstack tests.

dgirardi commented:
@carlosfelix this conflicts with https://github.com/prebid/Prebid.js/pull/11563 - and it doesn't look trivial to resolve. Could you submit a PR merging master into 9.0?

carlosfelix commented:
sure, working on it.

carlosfelix commented:
@dgirardi I've created the PR that merges master into 9.0 and solves the conflicts. https://github.com/prebid/Prebid.js/pull/11715
